src/customMetadata/GS_Admin_Checklist_Item.NPSP_Guide.md

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomMetadata xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <label>NPSP Guide</label>
+    <protected>true</protected>
+    <values>
+        <field>Description_Label__c</field>
+        <value xsi:type="xsd:string">gsChecklistItemNPSPGuideDesc</value>
+    </values>
+    <values>
+        <field>Extra_Info_Label__c</field>
+        <value xsi:nil="true"/>
+    </values>
+    <values>
+        <field>GS_Admin_Checklist_Section__c</field>
+        <value xsi:type="xsd:string">Make_it_your_own</value>
+    </values>
+    <values>
+        <field>Has_Link__c</field>
+        <value xsi:type="xsd:boolean">false</value>
+    </values>
+    <values>
+        <field>Image__c</field>
+        <value xsi:type="xsd:string">guide_illustration_1.svg</value>
+    </values>
+    <values>
+        <field>Link_Label__c</field>
+        <value xsi:nil="true"/>
+    </values>
+    <values>
+        <field>Link_URL__c</field>
+        <value xsi:nil="true"/>
+    </values>
+    <values>
+        <field>Position__c</field>
+        <value xsi:type="xsd:double">2.0</value>
+    </values>
+    <values>
+        <field>Primary_Button_Label__c</field>
+        <value xsi:type="xsd:string">gsChecklistItemNPSPGuidePriBtnLabel</value>
+    </values>
+    <values>
+        <field>Primary_Button_Type__c</field>
+        <value xsi:type="xsd:string">link</value>
+    </values>
+    <values>
+        <field>Primary_Button_Value__c</field>
+        <value xsi:type="xsd:string">https://power-of-us-hub-doc.s3-us-west-2.amazonaws.com/NPSP_Planning_Guide.pdf</value>
```

Comment:
@smarovich Is this a valid docs URL or should this point something without the Amazon values in the URL?

Reply:
@force2b say this url in this ticket https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008IEeaIAG/view